.github/scripts/cleanup-long-running-cluster.sh

```diff
@@ -49,13 +49,16 @@ fi
 
 # Delete all test namespaces.
 echo "delete all test namespaces"
-namespaces=$(kubectl get namespace |
```

Comment:
IMO this system is too complicated. Instead I think it's better to specify the namespaces we want to keep and delete everything else. Looks like there's already been some discussion about this: https://github.com/radius-project/radius/issues/7966